vmdb/app/views/layouts/_edit_to_email.html.haml

```diff
@@ -12,9 +12,11 @@
             = _("None")
           - else
             - if @temp[:email_to].present?
-              - emails = @temp[:email_to].sort { |a, b| a[1] <=> b[1] }.collect { |e| e[0] }
+              - emails = @temp[:email_to]
+                  .sort_by { |_email, display_email| display_name }
+                  .collect { |email, _display_email| email }
```

Comment:
A lot of the views use arbitrary variable names, but it's impossible to know what those are.  I've updated these to have proper variable names based on the data I see set in the controllers.  This makes it clear what the code is actually doing.